python/samples/demos/process_with_dapr/app.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import logging
+from contextlib import asynccontextmanager
+from enum import Enum
+from typing import TYPE_CHECKING, ClassVar
+
+import uvicorn
+from dapr.actor import ActorId
+from dapr.actor.runtime.context import ActorRuntimeContext
+from dapr.ext.fastapi import DaprActor, DaprApp
+from fastapi import FastAPI
+from fastapi.responses import JSONResponse
+from pydantic import Field
+
+from semantic_kernel import Kernel
+from semantic_kernel.functions import kernel_function
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+from semantic_kernel.processes.dapr_runtime.actors.event_buffer_actor import EventBufferActor
```

Comment:
```suggestion
from semantic_kernel.processes.dapr_runtime import EventBufferActor
```

Reply:
Adjusted for Dapr, local runtime, and other processes imports